docs/conveyor/user-guide.md

```diff
@@ -0,0 +1,68 @@
+# Getting started
+## Preparation
+You need to make sure that your wallet is funded with some DAI and USDC tokens. If you don't have the necessary tokens yet, follow the steps below to get funded.
+
+1. Get some test ETH from the [ropsten faucet](https://faucet.ropsten.be/).
+2. Swap ETH for USDC and DAI tokens on [uniswap](https://app.uniswap.org/#/swap) to swap DAI and USDC. Don't forget to connect Metamask to the ropsten test network.
```

Comment:
Oops it looks like I made a mistake here too. the "to swap DAI and USDC" is repetitive.